awscli/customizations/wizard/wizards/configure/_main.yml

```diff
@@ -0,0 +1,113 @@
+version: "0.1"
+title: Configure the AWS CLI
+description: This wizard will create a new CLI config profile for you
+plan:
+  decide_profile_type:
+    values:
+      config_type:
+        type: prompt
+        description: "What would you like to configure"
+        choices:
+          # There's also Web Identity and SAML but we're skipping that.
+          - display: Static Credentials
+            actual_value: static_creds
+          - display: Assume Role
+            actual_value: assume_role
+          - display: Process Provider
+            actual_value: proc_provider
+          - display: Additional CLI configuration
+            actual_value: additional_config
+      profile_name:
+        type: prompt
+        description: Enter the name of the profile
+    next_step:
+      switch: config_type
+      static_creds: get_static_creds
+      assume_role: get_assume_role
+      proc_provider: get_process_provider
+      additional_config: get_extra_cli_config
+  get_static_creds:
+    values:
+      access_key_id:
+        type: prompt
+        description: Enter your Access Key Id
+      secret_key:
+        type: prompt
+        description: Enter your Secret Access Key
+    next_step: DONE
```

Comment:
Running this option, I get the following:
```
$ aws configure wizard
What would you like to configure
Enter the name of the profile: blah                                                                                        
Enter your Access Key Id: dsfd                                                                                             
Enter your Secret Access Key: dsd                                                                                          

'role_arn'

```

Reply:
Yeah only the assume role portion is implemented.  I'll get the other parts fleshed out real quick.